app/build.gradle

```diff
@@ -558,6 +556,10 @@ tasks.whenTaskAdded { task ->
             && "true" == isConsumerApp && "true" == runDownloadScripts) {
         task.dependsOn downloadRestoreFile
     }
+    if ((task.name != null) && (task.name.startsWith('assemble') && task.name.endsWith('AndroidTest'))) {
```

Comment:
can't we use exact task name here ?

Reply:
because of the different `flavours` and build types we have a few combinations:
![image](https://github.com/user-attachments/assets/6d12c3b2-acd5-475a-98c4-b6cbd1b7467d)

Reply:
but think we only want to do this for `assebleCommcareReleaseAndroidTest` as that's the apk that gets passed to Browserstack ultimately.

Reply:
True and also, it seems that LTS is not relevant anymore. But what about CCConnect?

Reply:
LTS is still relevant though we have not lately pushed an update there (blocked on Android OS update). Though other flavors don't matter as we do all instrumentation testing on the main production flavour.